### ISSUE TITLE
hpfeeds error

### ISSUE THREAD

**Issue by dpicollege** [DinoTools/dionaea#95]
I config hpfeeds but I have this error on Ubuntu 14.04

[30032017 11:22:09] hpfeeds dionaea/hpfeeds.py:184-debug: hpclient msg opcode 0 data bytearray(b'identfail.')
[30032017 11:22:09] hpfeeds dionaea/hpfeeds.py:196-debug: errormessage from server: bytearray(b'identfail.')

could you explain how can I use hpfeeds modules. I have hpfeeds broker and every things is ok for server.

**Comment by phibos** [comment]
> hpfeeds error

???

> How is it possible to add new port to honeypot

- You can reconfigure an existing service to listen on the port you like it to liston on.
- You can use the [blackhole service](https://dionaea.readthedocs.io/en/latest/service/blackhole.html)
- You can write your own service. Feel free to have a look at the existing services as an example

> what is the mirror list and how i found out what is mirror and why we need this

Do you mean the mirror service?

**Comment by dpicollege** [comment]
yes mirror service and please add somethings like libemu to catch 64bit malware.
there is project that add libemu and unicorn https://github.com/fireeye/unicorn-libemu-shim
may be it's help you thanks in advance
